new/LIT14518QeneOzyan.xml

```diff
@@ -0,0 +1,82 @@
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="BNFabb145"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Qǝne with an allusion to the fall of Uzziah as described in <ref type="work" corresp="LIT3500Chroni">2 Chr. 26</ref> and 
+                    other biblical books. The type and author are not given in the manuscript. The verses end in ላ፡ and are marked with ።, ending up in 
+                    a series of possibly nine verses, so that the poem can be attributed the kwǝllǝkǝmu type of qǝne poetry according to the 
+                    classification in <bibl><ptr target="bm:Guidi1900Qene"/><citedRange unit="page">4-5</citedRange></bibl>  
+                    including two verse markers in <ref target="l2"/> and <ref target="l2"/>, which are not visible in the available pictures because 
+                    of the tight binding of the manuscript). </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Kwellekomu"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-11-01">Created entity.</change>
+            <change who="CH" when="2024-11-01">Updated edition with several readings supplied by Nafisa Valieva.</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="BNFabb145"/> on f. 55vc. Division into verses is indicated with ። . 
+                    The end of the poem is indicated with ። ። .</note>
+                <ab>
+                    <l n="1">ዖዝያን፡ ወድቀ፡ እመንበረ፡ ተድላ፡ አመ፡ ፈቀደ፡ ይፀዓን፡ ካህነተ፡ አሮን፡ ሰረገላ።</l> 
+                    <l n="2">እንዘ፡ ዲበ፡ ፈረስ፡ ቅይሕ፡ መንግሥተ፡ ገሊላ<supplied reason="undefined" resp="CH">።</supplied></l>
+                    <l n="3">ወአመ፡ ፈቀደ፡ በ<del rend="strikethrough"><unclear>ቅ</unclear>ንሣ<unclear>አ፡</unclear></del>ዝንጋዔ፡ ያስተጋብእ፡ 
+                        ፍሬ፡ ምህርካሁ፡ ለአማሌቅ፡ ሰግላ።</l>
+                    <l n="4">ሳኦል፡ ዘገልገላ።</l>
+                    <l n="5">እምላዕለ፡ ርእሱ፡ አውረደ፡ ትርሲተ፡ መንግሥቱ፡ ቀጸላ።</l>   
+                    <l n="6">አባግዕ፡ ጊዜ፡ መብልዕ፡ ከመ፡ ኢይሬእዩ፡ ተኵላ<supplied reason="undefined" resp="CH">።</supplied></l>
```

Comment:
You can just take the entire work ተኵላ into brackets for `<supplied/>`. On the regular image it is not visible well. The images taken by Nafisa stop exactly at that point (that is normal :)), in the dictionary it is indeed ተኵላ. ትኵላ might be a mistake of the copyist. We can consider this undecided, just supply the correct reading.

Reply:
OK, now I understand!